src/features/accounts/components/accounts-menu/hardware-authenticator/ledger-safari-warning.tsx

```diff
@@ -0,0 +1,15 @@
+import { Alert, AlertIcon, AlertProps, Text } from "components"
+
+export function LedgerSafariWarning(props: AlertProps) {
+  const userAgent = navigator.userAgent
+  const isSafari = userAgent.includes("Safari") && !userAgent.includes("Chrome")
```

Comment:
Non blocking: It could be simpler to check that at the root level and define `LedgerSafariWarning` to whether the real impl or return null.